bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZTo3LNu01j_5f_NLO_FXFX/WZTo3LNu01j_5f_NLO_FXFX_proc_card.dat

```diff
@@ -0,0 +1,21 @@
+#import model loop_sm-ckm_no_b_mass
```

Comment:
please remove commented import and follow in general the suggestions in 
https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T